shell/browser/browser_win.cc

```diff
@@ -369,4 +371,37 @@ void Browser::ShowEmojiPanel() {
   ::SendInput(4, input, sizeof(INPUT));
 }
 
+void Browser::ShowAboutPanel() {
+  const auto& opts = about_panel_options_;
+  std::string aboutMessage = "";
+  gfx::ImageSkia image;
+  const std::string* str;
+  std::vector<std::string> stringOptions = {
+      "applicationName", "applicationVersion", "copyright", "credits"};
+
+  if (opts.is_dict()) {
+    for (std::string opt : stringOptions) {
+      if ((str = opts.FindStringKey(opt))) {
+        aboutMessage.append(*str).append("\r\n");
+      }
+    }
+
+    if ((str = opts.FindStringKey("iconPath"))) {
+      base::FilePath path = base::FilePath::FromUTF8Unsafe(*str);
+      electron::util::PopulateImageSkiaRepsFromPath(&image, path);
+    }
+  }
+
+  electron::MessageBoxSettings settings = {};
+  settings.title = "About";
```

Comment:
Hardcoding the title won't let users localize their app - can we get around this problem?

Reply:
I think the easiest way would be to omit the "About" string entirely. This would be a cross-platform change, since it is omitted on macOS and Linux.

However, I don't have too much context on if that's a correct native Windows pattern.

Reply:
we can also use the l10n utility, potentially!

Reply:
Looked a bit through the l10n stuff, but didn't seem to find any strings that would match in here: https://cs.chromium.org/chromium/src/ui/strings/translations/ui_strings_en-GB.xtb

🤔 think i might just stick to removing it

Reply:
in that case, that's fine imo!